android/MobiledgeXSDKDemo/README.md

```diff
@@ -28,22 +28,23 @@ You can also choose this cloudlet to be used as the **Edge** server for Face Det
 
 ### SDK Calls
 #### Register Client
-This will call the [RegisterClientAPI](http://swagger.mobiledgex.net/client/#operation/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
+This will call the [RegisterClientAPI](https://
+.mobiledgex.net/#section/Edge-SDK-Android/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
 
 #### Get App Instances
-This will call the [GetAppInstListAPI](http://swagger.mobiledgex.net/client/#operation/GetAppInstList "GetAppInstList") to find everywhere our backend is running, and will draw a cloudlet icon for every location found.
+This will call the GetAppInstListAPI to find everywhere our backend is running, and will draw a cloudlet icon for every location found.
 
 #### Verify Location
-This will call the [VerifyLocationAPI](http://swagger.mobiledgex.net/client/#operation/VerifyLocation "VerifyLocation") with our current GPS coordinates to verify that the mobile device is where it claims to be. If successful, the mobile phone icon is green showing the accuracy information briefly. If location verification fails, the icon will be red, displaying the result code temporarily. Tapping the icon will show the result code of the call.
+This will call the [VerifyLocationAPI](https://swagger.mobiledgex.net/#section/Edge-SDK-Android/VerifyLocation "VerifyLocation") with our current GPS coordinates to verify that the mobile device is where it claims to be. If successful, the mobile phone icon is green showing the accuracy information briefly. If location verification fails, the icon will be red, displaying the result code temporarily. Tapping the icon will show the result code of the call.
```

Comment:
i think you wanted these links to point to https://api.mobiledgex.net/ ? I am getting a 404 from swagger.mobiledgex.net